### DISCUSSION TITLE
getting error when enabling wallbash

### DISCUSSION THREAD

**Discussion by vyrx-dev** [prasanthrangan/hyprdots#1067] [Q&A] [answered]
![240311_23h19m14s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/139481531/76ba9d5f-37b6-4631-a8f7-467b72594457)

when i enable wallbash by pressing win+shift+D it's giving me this error but before the update, it was working

**Comment by kRHYME7** [comment]
Rebuild your wallpaper cache.

``` Hyprdots cache rebuild ```


would take around 10-30 minutes depending on cpu.

**Reply by vyrx-dev** [reply]
done but still getting the same error

**Reply by kRHYME7** [reply, marked as answer]
Oh I updated the post You should remove the old ./scripts and ./wallbash directory  
```
rm -fr ~/.config/hypr/scripts  
rm -fr ~/.config/hypr/wallbash 
```

then turn on wallbash.

**Reply by vyrx-dev** [reply]
thanks
